toolkit/reg.sh

```diff
@@ -0,0 +1,358 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -x
+REG_HOME="$(
+  cd "$(dirname "$0")"
+  pwd
+)"
+REG_HOSTS=("core-1-1" "core-1-2" "core-1-3" "core-1-4" "core-1-5" "core-1-6" "core-1-7" "core-1-8")
+REG_CONF=${REG_HOME}/conf
+REG_TOOLS=${REG_HOME}/tools
+REG_SCRIPTS=${REG_HOME}/scripts
+REG_CELEBORN_DIST=${REG_HOME}/dist
+
+HIVEBENCH_DIR=${REG_HOME}/hive-testbench
+HIVEBENCH_QUERY_DIR=${HIVEBENCH_DIR}/spark-queries-tpcds
+
+HIBEN_DIR=${REG_HOME}/hibench3
+HIBEN_CONF_DIR=${HIBEN_DIR}/conf
+
+CHECK_TPCDS_PYTHON=${REG_SCRIPTS}/check.py
+
+CELEBORN_CONF_DIR=${REG_CONF}/celeborn
+RSS_INSTALL_DIR=/opt/apps/RSS/rss-0.1.1
```

Comment:
CELEBORN?

Reply:
The current installation still uses the old name "RSS".

Reply:
I guess we already changed all `RSS_` prefixes to `CELEBORN_` of env vars in the codebase, am I missed something?

Reply:
Forget about it, I'll rename it and the product name will be renamed at the next release.